src/styles/layout/_component-grid.scss

```diff
@@ -85,16 +78,25 @@ $breakpoint-accordion-large-755: 47rem;
     align-items: flex-start;
     color: var(--font-color-dark);
 
+    img {
+      object-position: bottom;
+      object-fit: scale-down;
+
+      @media (min-width: $breakpoint-scale-up) {
+        object-fit: contain;
```

Comment:
I don't think I've ever used `object-fit: scale-down` before, playing in the inspector it seems like `scale-down` might work at a larger breakpoint too (I can't see any visual difference in the images if I toggle between `scale-down` and `contain` but maybe I'm missing something), have you looked into this at all?

Reply:
`scale-down` was helping those images not grow outside of the component card. Otherwise, if I didn't change anything else (or swapped `scale-down` for `contain`/`cover`), I was getting something like this: 

<img width="679" alt="Screen Shot 2022-12-01 at 6 53 05 AM" src="https://user-images.githubusercontent.com/69602589/205046294-916673b3-cf6c-47a7-a64e-82ee17492251.png">

Reply:
I think `scale-down` is a really smart choice, since it chooses either `contain` or `none` for `object-fit` based on whatever works best. I'm wondering if this whole media query can be removed so that only `scale-down` is used?
```css
      @media (min-width: $breakpoint-scale-up) {
        object-fit: contain;
      }
```

Reply:
Oh gosh, sorry I misunderstood. Yes. That totally works! I think that was a remnant from when I was experimenting. Thanks for catching it!

Reply:
Also- I fixed it in this commit: https://github.com/sparkbox/accessible-components/pull/110/commits/349c925d4c76206ec981764a251703bd81f27551